assets/benchmarks/shaders/Benchmark_Texture.hlsl

```diff
@@ -14,7 +14,7 @@
 
 #include "VsOutput.hlsli"
 
-Texture2D Tex0 : register(t0);
+Texture2D Tex0 : register(t1);
```

Comment:
why do we need to change this to slot 1?

Reply:
could you add some comments here to explain why 1 is used? (in case some one want to make modifications in the future. )